src/validation.h

```diff
@@ -643,6 +649,8 @@ class CChainState
     void PruneAndFlush();
 
     /**
+     * Find the best known block, and make it the tip of the block chain
```

Comment:
There's still some redundant text in here. Consider:

```diff
diff --git a/src/validation.h b/src/validation.h
index 10943b6c24..2e1c441ba5 100644
--- a/src/validation.h
+++ b/src/validation.h
@@ -649,18 +649,16 @@ public:
     void PruneAndFlush();
 
     /**
-     * Find the best known block, and make it the tip of the block chain
-     *
-     * Make the best chain active, in multiple steps. The result is either failure
-     * or an activated best chain. pblock is either nullptr or a pointer to a block
-     * that is already loaded (to avoid loading it again from disk).
+     * Find the best known block, and make it the tip of the block chain. The
+     * result is either failure or an activated best chain. pblock is either
+     * nullptr or a pointer to a block that is already loaded (to avoid loading
+     * it again from disk).
      *
      * ActivateBestChain is split into steps (see ActivateBestChainStep) so that
      * we avoid holding cs_main for an extended period of time; the length of this
      * call may be quite long during reindexing or a substantial reorg.
      *
-     * May not be called with cs_main held. May not be called in a
-     * validationinterface callback.
+     * May not be called in a validationinterface callback.
      *
      * @returns true unless a system error occurred
      */
```

Reply:
Fixed in: d0454bad30253e735254cf7be359212293d952af